readme.md

```diff
@@ -206,3 +203,6 @@ yarn add jotai react
 - Advanced Recipes
   - [Large Objects](./docs/advanced-recipes/large-objects.mdx)
   - [Atom Creators](./docs/advanced-recipes/atom-creators.mdx)
+- Demo
+  - [Comparison](./docs/basics/comparison.mdx)
+  - [Showcase](./docs/basics/showcase.mdx)
```

Comment:
@sandren what do you think about moving them down here?
I expected many people are interested in "comparison" and "showcase" before diving into detailed docs.

Reply:
@dai-shi We can move the Demo Sub-section to the top of basics, I wouldn't mind at all

Reply:
In this PR, let's move them between Basics and Guides.
@sandren will probably re-organize anyway in follow-up PRs.

Reply:
done: 5d3d203